test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -26,29 +27,25 @@
 from ...integration.sagemaker.timeout import timeout
 
 MULTI_GPU_INSTANCE = 'ml.p3.8xlarge'
-
+RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
 
 def validate_or_skip_smmodelparallel(ecr_image):
     if not can_run_smmodelparallel(ecr_image):
         pytest.skip("Model Parallelism is supported on CUDA 11 on PyTorch v1.6 and above")
 
+def validate_or_skip_smmodelparallel_efa(ecr_image):
+    if not can_run_smmodelparallel_efa(ecr_image):
+        pytest.skip("Model Parallelism with EFA is supported on CUDA 11.1 on PyTorch v1.8 and above")
 
-def validate_or_skip_smdataparallel(ecr_image):
-    if not can_run_smdataparallel(ecr_image):
-        pytest.skip("Data Parallelism is supported on CUDA 11 on PyTorch v1.6 and above")
-
-
-def can_run_smdataparallel(ecr_image):
+def can_run_smmodelparallel(ecr_image):
     _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
     image_cuda_version = get_cuda_version_from_tag(ecr_image)
     return Version(image_framework_version) in SpecifierSet(">=1.6") and Version(image_cuda_version.strip("cu")) >= Version("110")
 
-
-def can_run_smmodelparallel(ecr_image):
+def can_run_smmodelparallel_efa(ecr_image):
```

Comment:
Not a part of review: I wonder if this could be a `LABEL` on the image that gets built.

Reply:
didnt get this

Reply:
Please ignore.